spec/features/genres/visitor_can_see_genres_but_not_form_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+describe "As an admin" do
```

Comment:
Your test file says as a visitor, but this description section says as an admin.